modular_splurt/code/modules/jobs/job_types/blueshield.dm

```diff
@@ -1,28 +1,26 @@
 /datum/job/blueshield
 	title = "Blueshield"
 	flag = BLUESHIELD
-	department_head = list("Central Command")
+	department_head = list("Head of Security")
 	department_flag = ENGSEC
 	faction = "Station"
-	head_announce = list(RADIO_CHANNEL_COMMAND)
-	supervisors = "central command"
-	total_positions = 1
-	spawn_positions = 1
+	supervisors = "Head of Security"
+	total_positions = 3
```

Comment:
The extra positions worry me, mostly due to you also increasing damage of the guns by 10.